templates/python/known.go

```diff
@@ -0,0 +1,7 @@
+package python
+
+const hostTpl = `
+`
+
+const emailTpl = `
```

Comment:
This is dangerous. If a validation is not supported, then attempting to use it should error loudly, not silently succeed.